### PR TITLE
fix: Add some missing Chinese translation content, including superset-ui project

### PR DESCRIPTION
**Pull request by cdmikechen**
### SUMMARY
Add some missing Chinese translation content, including superset-ui project. 
Some of the current Chinese translations are wrong. At present, most translation errors have been fixed, and then some missing parts and `superset-ui` project missing parts have been added.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
databaselist

### TEST PLAN
The result of translation can be viewed in the browser

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 1]
```diff
@@ -196,38 +222,52 @@
...
-      "Deleted %(num)d chart": [""],
-      "Request is not JSON": [""],
-      "Request is incorrect: %(error)s": [""],
-      "`confidence_interval` must be between 0 and 1 (exclusive)": [""],
+      "Name must be unique": ["名称必须是唯一的"],
+      "Deleted %(num)d chart": ["删除了 %(num)d 个图表"],
+      "Request is not JSON": ["请求不是JSON"],
+      "Request is incorrect: %(error)s": ["请求不正确: %(error)s"],
+      "`confidence_interval` must be between 0 and 1 (exclusive)": [
+        "`置信区间` 必须介于0和1之间（独占）"
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:230]:
```suggestion
        "`置信区间` 必须介于0和1之间（开区间）"
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 2]
```diff
@@ -196,38 +222,52 @@
...
-      "Owners are invalid": [""],
+      "Import chart failed for an unknown reason": ["导入图表失败，原因未知"],
+      "Owners are invalid": ["所有者无效"],
       "Dataset does not exist": ["数据集不存在"],
-      "`operation` property of post processing object undefined": [""],
-      "Unsupported post processing operation: %(operation)s": [""],
-      "Adding new datasource [{}]": ["添加 Druid 数据源"],
-      "Refreshing datasource [{}]": [""],
+      "`operation` property of post processing object undefined": [
+        "处理对象的 `操作` 属性未定义"
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:264]: @ktmud this line isn't very clear, can you help?

Comment (ktmud) [reply superset/translations/zh/LC_MESSAGES/messages.json:264]:
```
后处理未指定操作类型（`operation`）
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 3]
```diff
@@ -403,223 +457,287 @@
...
+      "`fbprophet` package not installed": ["未安装程序包 `fbprophet`"],
+      "Time grain missing": ["时间粒度缺失"],
+      "Unsupported time grain: %(time_grain)s": [
+        "不支持的时间粒度：%(time_grain)s"
+      ],
+      "Periods must be a positive integer value": ["句点必须是正整数值"],
+      "Confidence interval must be between 0 and 1 (exclusive)": [
+        "置信区间必须介于0和1（不包含1）之间"
+      ],
+      "DataFrame must include temporal column": ["数据集必须包含时间列"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:693]: any reasons why DataFrame has different translations?

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 4]
```diff
@@ -1137,418 +1288,444 @@
...
-      "Check out this chart in dashboard:": ["查看这个仪表板：%s"],
+      "Check out this chart in dashboard:": ["查看这个看板：%s"],
       "Select ...": ["选择 ..."],
       "Loaded data cached": ["数据缓存已加载"],
       "Loaded from cache": ["从缓存中加载"],
       "Click to force-refresh": ["点击强制刷新"],
-      "cached": ["缓存于%s"],
-      "Certified by %s": [""],
+      "cached": ["已缓存"],
+      "Certified by %s": ["认证 %s"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:1364]: "由 %s认证" ？

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 5]
```diff
@@ -1137,418 +1288,444 @@
...
       "delete": ["删除"],
-      "Type \"%s\" to confirm": [""],
-      "DELETE": [""],
-      "Click to edit": ["点击编辑标题"],
+      "Type \"%s\" to confirm": ["键入 \"%s\" 来确认"],
+      "DELETE": ["删除"],
+      "Click to edit": ["点击编辑"],
       "You don't have the rights to alter this title.": [
-        "你没有权利改变这个标题。"
+        "你没有权利修改这个标题。"
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:1384]: 为保一致性，可以通用敬称“您”吗？

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 6]
```diff
@@ -1137,418 +1288,444 @@
...
-      "Apply to all panels": [""],
-      "Apply to specific panels": [""],
-      "Only selected panels will be affected by this filter": [""],
-      "All panels with this column will be affected by this filter": [""],
-      "All filters": ["过滤"],
-      "All charts": ["没有图表"],
+      "(Removed)": ["(已删除)"],
+      "Undo?": ["撤消？"],
+      "Scoping": ["范围"],
+      "Apply to all panels": ["适用于所有面板"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:1602]:
```suggestion
      "Apply to all panels": ["应用于所有面板"],
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 7]
```diff
@@ -1137,418 +1288,444 @@
...
+      "Apply to specific panels": ["适用于特定面板"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:1603]:
```suggestion
      "Apply to specific panels": ["应用于特定面板"],
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 8]
```diff
@@ -1137,418 +1288,444 @@
...
-      "Metadata has been synced": [""],
-      "Column name [%s] is duplicated": [""],
-      "Metric name [%s] is duplicated": [""],
-      "Calculated column [%s] requires an expression": [""],
-      "Basic": [""],
-      "Default URL": ["数据库URL"],
+        "来确保字符的表达顺序与时间顺序一致的标准。如果时间戳格式不符合 ISO 8601 标准，则需要定义表达式和类型，以便将字符串转换为日期或时间戳。注意：当前不支持时区。如果时间以epoch格式存储，请输入 `epoch_s` or `epoch_ms` 。如果没有指定任何模式，我们可以通过额外的参数在每个数据库/列名级别上使用可选的默认值。"
+      ],
+      "Is dimension": ["维度"],
+      "Is filterable": ["过滤条件"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:1637]:
```suggestion
      "Is filterable": ["可被过滤"],
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 9]
```diff
@@ -1819,381 +2004,1116 @@
...
-        "获取表格元数据时发生错误"
+        "获取数据集相关数据时出错"
       ],
       "An error occurred while fetching dataset related data: %s": [
-        "获取表格元数据时发生错误"
+        "获取数据集相关数据时出错：%s"
       ],
-      "Physical dataset": ["选择数据源"],
-      "Virtual dataset": ["编辑数据库"],
+      "Physical dataset": ["物理数据集"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:2321]: 用物化会不会高级些

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 10]
```diff
@@ -1819,381 +2004,1116 @@
...
-      "Type or Select [%s]": ["选择 [%s]"],
-      "Filter box": ["筛选盒"],
-      "Filters configuration": ["配置Layer"],
-      "Filter configuration for the filter box": [""],
+      "Share": ["分享"],
+      "Last run %s": ["上次执行 %s"],
+      "Recents": ["最近"],
+      "Select start and end date": ["选择开始和结束时间"],
+      "Type or Select [%s]": ["键入或选择 [%s]"],
+      "Filter box": ["过滤选择器"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:2419]:
```suggestion
      "Filter box": ["过滤器"],
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 11]
```diff
@@ -1819,381 +2004,1116 @@
...
+        "D3时间插件格式语法: https://github.com/d3/d3-time-format"
+      ],
+      "D3 format syntax: https://github.com/d3/d3-format": [
+        "D3插件格式语法: https://github.com/d3/d3-time-format"
+      ],
+      "Whether to apply filter to dashboards when table cells are clicked": [
+        "单击表单元格时是否对看板应用过滤条件"
+      ],
+      "Adaptative formating": ["自动匹配格式化"],
+      "Aggregate": ["合计"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:2617]:
```suggestion
      "Aggregate": ["聚合"],
```

File: superset/translations/zh/LC_MESSAGES/messages.json [hunk 12]
```diff
@@ -1137,418 +1288,444 @@
...
-      "Are you sure you want to save and apply changes?": [""],
-      "Confirm save": [""],
-      "Edit Dataset ": ["编辑数据库"],
-      "Use legacy datasource editor": [""],
-      "Time range": ["时间的粒度"],
+      "Confirm save": ["确认保存"],
+      "Edit Dataset ": ["编辑数据集"],
+      "Use legacy datasource editor": ["使用旧数据源编辑器"],
+      "Time range": ["时间范围"],
       "Time column": ["时间字段"],
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.json:1715]: time column 时间维度

File: superset/translations/zh/LC_MESSAGES/messages.po [hunk 13]
```diff
@@ -164,75 +164,75 @@ msgstr "刷新 Druid 元数据"
...
 
 #: superset/errors.py:110
 msgid "Issue 1006 - One or more parameters specified in the query are missing."
-msgstr ""
+msgstr "Issue 1006 - 查询中指定的一个或多个参数丢失。"
 
 #: superset/databases/schemas.py:168 superset/exceptions.py:125
 msgid "Invalid certificate"
-msgstr ""
+msgstr "无效证书"
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.po:197]:
```suggestion
msgstr "无效认证"
```

File: superset/translations/zh/LC_MESSAGES/messages.po [hunk 14]
```diff
@@ -3468,48 +3505,48 @@ msgstr "创建数据源，并弹出一个新的标签页"
...
 #: superset-frontend/src/SqlLab/components/ExploreResultsButton.jsx:156
 msgid "Explore the result set in the data exploration view"
-msgstr ""
+msgstr "在数据探索视图中探索结果集"
 
 #: superset-frontend/src/SqlLab/components/ExploreCtasResultsButton.jsx:99
 #: superset-frontend/src/SqlLab/components/ExploreResultsButton.jsx:163
 msgid "Explore"
-msgstr "探索图表"
+msgstr "探索"
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.po:3518]:
```suggestion
msgstr "Explore"
```

File: superset/translations/zh/LC_MESSAGES/messages.po [hunk 15]
```diff
@@ -3665,23 +3708,43 @@ msgstr "运行选定的查询"
...
 
 #: superset-frontend/src/SqlLab/components/SaveDatasetModal.tsx:108
 msgid "Save & Explore"
-msgstr "保存一个切片"
+msgstr "保存和浏览"
 
 #: superset-frontend/src/SqlLab/components/SaveDatasetModal.tsx:123
 msgid "Overwrite & Explore"
-msgstr "覆盖切片 %s"
+msgstr "覆盖和浏览"
```
Comment (junlincc) [inline superset/translations/zh/LC_MESSAGES/messages.po:3727]: 建议用 覆写

**Comment by junlincc** [comment]
@cdmikechen Thanks for the contribution, i learned so much by going through your translated file. 
couple asks~ 
can you also share a glossary if you do have, so we can help build upon? also do you mind reaching out to me in Slack or email(junlin@preset.io), I would love to understand how you are currently using Superset, as well as some specific user needs in China. Thanks!

**Comment by cdmikechen** [comment]
@junlincc Hi~
I've changed the part that you reviewed.

**Review by junlincc — APPROVED** [review]
Approved, and thank you Xiang, for contributing back to the community! @cdmikechen 

We added a related roadmap item, https://github.com/apache-superset/superset-roadmap/issues/153 
Let us know if you are interested in helping :) We look forward to work with you more!

**Review by ktmud — COMMENTED** [review]
Sorry for being late to the party, but here are a couple of more suggestions that I never had time to finish.

Feel free to ignore those since I'm sure the merged translations are already much better than what we had before.

**Pull request merged**